ghost/core/core/frontend/services/data/checks.js

```diff
@@ -3,6 +3,11 @@ function isPost(jsonData) {
         Object.prototype.hasOwnProperty.call(jsonData, 'title') && Object.prototype.hasOwnProperty.call(jsonData, 'slug');
 }
 
+function isNewsletter(jsonData) {
+    return Object.prototype.hasOwnProperty.call(jsonData, 'name') &&
+        Object.prototype.hasOwnProperty.call(jsonData, 'subscribe_on_signup') && Object.prototype.hasOwnProperty.call(jsonData, 'slug');
```

Comment:
Maybe we could use less overlapping named fields here? How about `subscribe_on_signup`, `status` and `visibility`?